src/main/resources/portal.properties.EXAMPLE

```diff
@@ -231,24 +231,23 @@ oncoprint.hotspots.default=true
 # Custom gene sets
 # querypage.setsofgenes.location=file:/<path>
 
-# to disable use of Ehcache, set ehcache.cache.enabled to false
-ehcache.cache_enabled=true
-ehcache.xml_configuration=/ehcache.xml
-ehcache.statistics_enabled=true
+# to enable use of Ehcache, set ehcache.cache.enabled to true and uncomment all Ehcache properties
```

Comment:
fix the property referenced in this comment - it's `ehcache.cache.enabled` instead of `ehcache.cache_enabled`

Reply:
👍